acoustid/future/api/app.py

```
@@ -0,0 +1,90 @@
+
+
+def create_app() -> Starlette:
+    app = Starlette(
+        routes=[
+            Route("/v3/submission", handle_submission, methods=["POST"]),
+        ],
+        exception_handlers={
+            ValidationError: handle_validation_error,
+        },
+    )
+    return app
```

Ruff (0.8.2)

Unnecessary assignment to `app` before `return` statement

Remove unnecessary assignment

(RET504)